recipes/dsdp/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "5.8" %}
+
+{% set variant = "openblas" %}
+
+
+package:
+  name: dsdp
+  version: {{ version }}
+
+source:
+  fn: DSDP{{ version }}.tar.gz
+  url: http://www.mcs.anl.gov/hs/software/DSDP/DSDP{{ version }}.tar.gz
+  sha256: 26aa624525a636de272c0b329e2dfd01a0d5b7827f1c1c76f393d71e37dead70
+  patches: dsdp_osx_patch.diff    # [osx]
+
+build:
+  skip: true    # [win32]
+  number: 200
+  features:
+    - blas_{{ variant }}
+    - vc14    # [win]
+
+
+requirements:
+  build:
+    - cmake
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.20|0.2.20.*
+
+  run:
+    - blas 1.1 {{ variant }}
+    - openblas 0.2.20|0.2.20.*
+
+test:
+  source_files:
+    - bin/control1.dat-s
+    - bin/mcp100.dat-s
+    - bin/truss1.dat-s
+  commands:
+    - dsdp5 bin/control1.dat-s   # [not win]
+    - dsdp5 bin/mcp100.dat-s   # [not win]
+    - dsdp5 bin/truss1.dat-s   # [not win]
+    - conda inspect linkages -p ${PREFIX} dsdp  # [not win]
+    - conda inspect objects -p ${PREFIX} dsdp  # [osx]
```

Comment:
Could we add some file tests as well given we are having to homebrew the build step? Linked an example of such a pattern below.

ref: https://github.com/conda-forge/hdf5-feedstock/blob/efa53d2c8fa4c00da4ca53a0a6df80f32159cc17/recipe/meta.yaml#L92-L105

Reply:
note: Done below.